tweb/src/main/frontend/src/js/vue/components/ProjectsOverview.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<div class="col-12 contained">
+		<div class="scrolling-wrapper-flexbox">
+			<project-card v-for="project in projects" :project="project" v-bind:key="project.name"></project-card>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+    import ProjectCard from "./ProjectCard.vue";
+	import ProjectOverviewDetail from "../models/ProjectOverviewDetail"
+	import axios from 'axios';
+
+	@Component({
+		components: {
+			'project-card': ProjectCard
+		}
+	})
+    export default class ProjectsOverview extends Vue {
+
+		private projectDataURL = "https://raw.githubusercontent.com/FundRequest/platform-metadata/master/projects"
+		private projects: Array<ProjectOverviewDetail> = []
+
+		private _loadProjects(projects) {
+
+			axios.get(this.projectDataURL)
+				.then(response => {
+					let projectData = response.data;
+					for (var proj in projectData) {
+						projects.push({
+							name: projectData[proj]['title'],
```

Comment:
To make the object of type ProjectOverviewDetail, you can use
        `Object.assign(new ProjectOverviewDetail(), { name: projectData[proj]['title'], ....... });`
Else there are errors thrown when passing this as a prop to project-card, since they expect type  `ProjectOverviewDetail`, but actually get type `object`.